File: JavaBasic/119_palindrome_num.java
// 119	119_palindrome_num.java	Check whether a given number is a palindrome (e.g. 121, 1331).
public class _119_palindrome_num {
    public static void main(String[] args) {
        int num = 1331, original = 1331, reversed = 0, temp = num;
        while (temp > 0) { reversed = reversed*10 + temp%10; temp /= 10; }
        System.out.println("Number: " + original);
        System.out.println(original == reversed ? "Palindrome." : "NOT a Palindrome.");
    }
}

File: JSBASIC/119_palindrome_num.js
// 119	119_palindrome_num.js	Check whether a given number is a palindrome (e.g. 121, 1331).

let num = 1331;
let original = num;
let reversed = 0;
let temp = num;
while (temp > 0) {
    reversed = reversed * 10 + (temp % 10);
    temp = Math.floor(temp / 10);
}
console.log("Number:", original);
if (original === reversed) {
    console.log(original, "is a Palindrome.");
} else {
    console.log(original, "is NOT a Palindrome.");
}
